doc/sphinx/introduction.rst

```diff
@@ -26,37 +26,35 @@ programs called *tactics*.
 All services of the |Coq| proof assistant are accessible by interpretation
 of a command language called *the vernacular*.
 
-Coq has an interactive mode in which commands are interpreted as the
+Coq has an interactive mode in which commands are interpreted as the
 user types them in from the keyboard and a compiled mode where commands
 are processed from a file.
 
--  The interactive mode may be used as a debugging mode in which the
-   user can develop his theories and proofs step by step, backtracking
-   if needed and so on. The interactive mode is run with the `coqtop` 
-   command from the operating system (which we shall assume to be some
-   variety of UNIX in the rest of this document).
+-  In interactive mode, users can develop their theories and proofs step by
+   step, and query the system for available theorems and definitions. The
+   interactive mode is generally run with the help of an IDE, such
+   as CoqIDE, documented in :ref:`coqintegrateddevelopmentenvironment`,
+   Emacs with Proof-General (available at https://proofgeneral.github.io/),
```

Comment:
Is there any reason to drop the proof-general citation?

Reply:
Given that the citation was basically just a link, I thought it would be much better to inline it in the text...

Reply:
Right; the proper citation is this:

```
@InCollection{ProofGeneral2000,
  Title        = {Proof General: A Generic Tool for Proof Development},
  Author       = {Aspinall, David},
  Booktitle    = {Tools and Algorithms for the Construction and
               Analysis of Systems, {TACAS} 2000},
  Publisher    = {Springer Berlin Heidelberg},
  Year         = {2000},
  Editor       = {Graf, Susanne and Schwartzbach, Michael},
  Pages        = {38--43},
  Series       = {Lecture Notes in Computer Science},
  Volume       = {1785},
  Doi          = {10.1007/3-540-46419-0_3},
  ISBN         = {978-3-540-67282-1},
  Url          = {http://dx.doi.org/10.1007/3-540-46419-0_3}
}
```

Reply:
Thanks, citation added.